src/coreclr/nativeaot/System.Private.TypeLoader/src/Resources/Strings.resx

```diff
@@ -1,17 +1,17 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <root>
-  <!-- 
-    Microsoft ResX Schema 
-    
+  <!--
```

Comment:
Forgive the autosave reformatting, I can revert the white-space changes if it matters.

Reply:
NIT: normally we revert unrelated changes